src/terminal_ui.cc

```diff
@@ -199,10 +214,10 @@ void TerminalUI::Screen::set_face(const Face& face)
     bool join = false;
     if (face.attributes != m_active_face.attributes)
     {
-        for (int i = 0; i < sizeof(attr_table) / sizeof(int); ++i)
+        for (int i = 0; i < sizeof(attr_table) / sizeof(attr_table[0]); ++i)
```

Comment:
we could use `std::size(attr_table)` here to avoid having to do the division dance.